RiotSwiftUI/Modules/Room/StaticLocationSharingViewer/Coordinator/StaticLocationSharingViewerCoordinator.swift

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import UIKit
+import SwiftUI
+import MatrixSDK
+
+struct StaticLocationSharingViewerCoordinatorParameters {
+    let roomDataSource: MXKRoomDataSource
+    let mediaManager: MXMediaManager
+    let avatarData: AvatarInputProtocol
+    let location: CLLocationCoordinate2D
+    let coordinateType: MXEventAssetType
+}
+
+final class StaticLocationSharingViewerCoordinator: Coordinator, Presentable {
```

Comment:
I'm not particularly happy with this name. Does the `StaticLocationViewing` prefix work better than `StaticLocationSharingViewer`? -> `StaticLocationViewingCoordinator`, `StaticLocationViewingModels`, `StaticLocationView/Screen` instead of `StaticLocationSharingViewer` etc.